src/lineAnalysis.js

```diff
@@ -0,0 +1,76 @@
+import * as cellValue from '../src/cellValue';
+import * as gameHelper from '../src/components/game/gameHelper';
+
+export function isThereAWinner(grid) {
+  let winningResults = {winnerFound:false, winningMove:null};
+  winningLines(grid).forEach(function(line) {
+    const [a, b, c] = line;
+    let potentialWinningLine = [grid[a], grid[b], grid[c]]
+    if (potentialWinningLine.every( cell => isThereAWinningLine(cell, potentialWinningLine)))
+      winningResults = {winnerFound:true, winningMove:potentialWinningLine[0]}
+  });
+  return winningResults;
+}
+
+export function isThereADraw(grid) {
+  let numberOfEmptyCells = gameHelper.countMark(cellValue.EMPTY, grid);
```

Comment:
I think this should be a const. I think Create-React-App includes a linter by default